txpool/txpool_test.go

```diff
@@ -20,6 +20,26 @@ var forks = &chain.Forks{
 	Istanbul:  chain.NewFork(0),
 }
 
+type mockStore struct {
+}
+
+func (m *mockStore) GetNonce(root types.Hash, addr types.Address) uint64 {
+	return 0
+}
+
+func (m *mockStore) GetBlockByHash(types.Hash, bool) (*types.Block, bool) {
+	return nil, false
+}
+
+func (m *mockStore) GetBalance(root types.Hash, addr types.Address) *big.Int {
```

Comment:
Nitpick, but you can omit both of those:
```suggestion
func (m *mockStore) GetBalance(types.Hash, types.Address) *big.Int {
```

Reply:
Fixed 👍 

6c05726